libraries/AP_Scripting/applets/VTOL-quicktune.lua

```diff
@@ -314,8 +320,12 @@ function update_slew_gain()
       slew_steps = slew_steps - 1
       logger.write('QUIK','SRate,Gain,Param', 'ffn', get_slew_rate(axis), P:get(), axis .. ax_stage)
       if slew_steps == 0 then
-         gcs:send_text(0, string.format("%s %.4f", slew_parm, P:get()))
-         slew_parm = nil
+         gcs:send_text(MAV_SEVERITY_INFO, string.format("%s %.4f", slew_parm, P:get()))
+      end
+      slew_parm = nil
```

Comment:
setting slew_parm to nil here breaks the slewing and will leave you with a parameter value that is too high. We should only set to nil when slew_steps have reached 0

Reply:
oops....I was testing and accidentally made the final edits on the actual script in the test folder instead of the local repo...sorry....had already corrected that...